webapp/src/components/modals/channel_subscriptions/channel_subscriptions.tsx

```diff
@@ -3,22 +3,27 @@
 
 import React, {PureComponent} from 'react';
 
-import {ProjectMetadata, ChannelSubscription} from 'types/model';
-
 import FullScreenModal from '../full_screen_modal/full_screen_modal';
 
+import {AllProjectMetadata} from 'types/model';
+
 import ChannelSubscriptionsModalInner from './channel_subscriptions_internal';
 import {SharedProps} from './shared_props';
 
 import './channel_subscriptions_modal.scss';
 
 export type Props = SharedProps;
 
-export default class ChannelSubscriptionsModal extends PureComponent<Props> {
+type State = {
+    showModal: boolean;
+    allProjectMetadata: AllProjectMetadata | null
```

Comment:
The core webapp eslint has a rule to enforce consistent syntax here, though not sure which rule applies that

```suggestion
    allProjectMetadata: AllProjectMetadata | null;
```